### PR TITLE
fix: missing breadcrumbs data on hardcrash

### PR DESCRIPTION
**Pull request by marandaneto**
also fixing automatic low memory breadcrumb action, so it matches with Android

### CODE REVIEW

File: Tests/Resources/breadcrumb.json [hunk 1]
```diff
@@ -0,0 +1,3139 @@
+{
```
Comment (bruno-garcia) [inline Tests/Resources/breadcrumb.json:1]: Can we trim down this a little? Rather large file

Comment (marandaneto) [reply Tests/Resources/breadcrumb.json:1]: @bruno-garcia I just copied from other sample and not sure what to trim, `fatalError` resource. sentry event itself has empty extra, tags and the breadcrumb, all the other things are about SentryCrash

Comment (bruno-garcia) [reply Tests/Resources/breadcrumb.json:1]: I'd remove anything that is not related to the `breadcrumb` (assuming based on the name) test case. The biggest offender there is the loaded images though

Comment (bruno-garcia) [reply Tests/Resources/breadcrumb.json:1]: `binary_images`

Comment (marandaneto) [reply Tests/Resources/breadcrumb.json:1]: yep it seems to be working, my 1st try failed on `isValidReport` and I just copied a sample, I was probably guessing it was a normal sentry event :D good catch.

**Pull request merged**